robot_skills/src/robot_skills/robot.py

```diff
@@ -126,7 +94,7 @@ def publish_target(self, x, y):
         self.pub_target.publish(geometry_msgs.msg.Pose2D(x, y, 0))
 
     def tf_transform_pose(self, ps, frame):
-        output_pose = geometry_msgs.msg.PointStamped
+        # output_pose = geometry_msgs.msg.PointStamped()
```

Comment:
Please remove this commented code!

Reply:
Restored old situation. As I am not sure what the possible outcomes are with TF. Also not really in the focus of this PR.